homework/homework_20/homework_20_2.py

```diff
@@ -0,0 +1,48 @@
+"""Homework # 20.2"""
+import unittest
+from decimal import Decimal
+import logging
+from homework.homework_11 import homework_11_2
+
+formatter = logging.Formatter('[%(asctime)s] %(levelname)s - %(message)s')
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+
+logger = logging.getLogger(__name__)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
```

Comment:
You can use 1 logger across an application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application